Library/Formula/docker-swarm.rb

```diff
@@ -0,0 +1,43 @@
+require "language/go"
+
+class DockerSwarm < Formula
+  homepage "https://github.com/docker/swarm"
+  url "https://github.com/docker/swarm/archive/v0.1.0.tar.gz"
+  version "0.1.0"
```

Comment:
A fully up-to-date brew create added the `version` I can remove it but I suggest you fix the `create` command otherwise everyone's formulas are going to have it.